src/PhpSpec/Container/ServiceLocator.php

```diff
@@ -148,12 +146,12 @@ public function get($id)
     }
 
     /**
-     * @param $id
+     * @param string $serviceId
      * @return bool
      */
-    public function isDefined($id)
+    public function has($serviceId)
```

Comment:
Add this for an alias in the first set of changes (until we are sure what happens with extensions)